### DISCUSSION TITLE
Feature request: show first release containing a merged pull request

### DISCUSSION THREAD

**Discussion by rossng** [community/community#21683] [New to GitHub]
<p>Just now, I was looking at <a href="https://github.com/NuGet/NuGet.Client/pull/2271" rel="nofollow noopener">this issue</a> and wanted to know if the feature merged there was available in the version of NuGet that I am using.</p>
<p>Currently, the only way to do this seems to be to manually make a note of the date and then compare it to the list of releases to figure out when the merged code went into production.</p>
<p>It would be very helpful for merged pull requests to display something that lets me immediately see ‘hey, this code went into release 4.8.2’ or ‘this fix hasn’t been released yet’.</p>
<p>(p.s. there doesn’t seem to be any official way to submit feature ideas to GitHub. I actually tried to tag this message ‘feature-request’ and got the message ‘The message tags contain feature, which is not permitted in this community.’ Maybe a separate forum section would be a good idea?)</p>

**Comment by BelHFL** [comment, marked as answer]
<p>Hi <a class="mention" href="/rossng">@rossng</a> ,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by why-el** [comment]
<p>Came here to make the same request. </p>
